pkgs/servers/search/elasticsearch/plugins.nix

```diff
@@ -6,6 +6,8 @@ let
     installPhase ? ''
       mkdir -p $out/config
       mkdir -p $out/plugins
+      mkdir -p $out/lib
+      cp -r ${elasticsearch-oss}/lib/* $out/lib/
```

Comment:
I am not sure if this has ever worked for newer versions. Should we backport? @basvandijk

Reply:
it does work with the previous version, doesn't really make sense to me but it does.

Reply:
Is @shmish111 symlinking the libraries possible?

Reply:
um, don't understand :-(

Reply:
I mean 'ln -s ${elasticsearch-oss}/lib/* $out/lib/'

Reply:
ah I see, we can, what are the advantages/disadvantages?

Reply:
The advantage would be less duplication.